CMakeLists.txt

```
@@ -11,14 +11,14 @@ project(
     kdmp-parser
     DESCRIPTION "A Cross-Platform C++ parser library for Windows kernel minidumps."
     HOMEPAGE_URL https://github.com/0vercl0k/kdmp-parser
-    VERSION 0.6.0
+    VERSION 0.7.0
 )
 
 set(PROJECT_AUTHOR 0vercl0k)
 set(PROJECT_LICENSE MIT)
 
 option(BUILD_PARSER "Build the parser executable for kdmp-parser" ON)
-option(BUILD_TESTS "Build the test suite for kdmp-parser" ON)
+option(BUILD_TESTS "Build the test suite for kdmp-parser" OFF)
 option(BUILD_PYTHON_BINDING "Build the Python bindings for kdmp-parser" ON)
```

Quick suggestion, set this option to `OFF` by default? Building the bindings requires more dependencies they may not want/need - and I assume most people who only want the bindings will go through PyPI.

```suggestion
option(BUILD_PYTHON_BINDING "Build the Python bindings for kdmp-parser" OFF)
```

---

Good catch!